packages/playground/windows/playground-win32/Playground-Win32.cpp

```diff
@@ -468,5 +468,5 @@ _Use_decl_annotations_ int CALLBACK WinMain(HINSTANCE instance, HINSTANCE, PSTR
   WINRT_VERIFY(classId);
   winrt::check_win32(!classId);
 
-  return RunPlayground(showCmd, true);
+  return RunPlayground(showCmd, false);
```

Comment:
i think you can remove the 2nd param and the assignment in 398, and just let the default-constructed WindowData set it will the default value, instead of overwriting it